### PR TITLE
Refactor user model

### PR DESCRIPTION
**Pull request by siaw23**

### CODE REVIEW

**Comment by milaaraujo** [comment]
Hey @siaw23, does this PR fix a specific issue?

**Comment by siaw23** [comment]
Nope @milaaraujo :) just making existing code better.

**Comment by gauravano** [comment]
Hey @siaw23, there an old [PR](https://github.com/publiclab/plots2/pull/2581) and as @jywarren mentioned here https://github.com/publiclab/plots2/pull/2581#issuecomment-451669220 to reuse the enum approach, what do you think about implementing it here? 

Thanks!

**Comment by siaw23** [comment]
I'm not sure I have time to do that now. If this PR is ready or you want some changes please let me know @gauravano @jywarren . Otherwise I'll let this be.

File: app/models/user.rb [hunk 1]
```diff
@@ -430,13 +439,4 @@ def self.count_all_time_contributor
     revisions = Revision.where(status: 1).pluck(:uid)
     contributors = (notes + answers + questions + comments + revisions).compact.uniq.length
   end
-
-  def self.to_csv(options = {})
```
Comment (jywarren) [inline app/models/user.rb]: Oh, what happened here?

Comment (siaw23) [reply app/models/user.rb]: This method is in `application_record.rb` as a class method. All records inherit it.

**Comment by jywarren** [comment]
Thanks @siaw23!!

**Pull request merged**